mongo/integration/json_helpers_test.go

```diff
@@ -405,17 +405,19 @@ func errorFromResult(t testing.TB, result interface{}) *operationError {
 func verifyError(t testing.TB, expected *operationError, actual error) {
 	t.Helper()
 
-	expectErr := expected != nil
```

Comment:
I changed this error handling because at some point I had an error that just said `expected error: false, got error: true` which is not helpful when debugging. The new code actually prints out an error if one was not expected.